src/components/earnings/RedeemHoloFuelCard.vue

```diff
@@ -0,0 +1,177 @@
+<script setup lang="ts">
+import { ChevronLeftIcon } from '@heroicons/vue/20/solid'
+import BaseButton from '@uicommon/components/BaseButton.vue'
+import BaseCard from '@uicommon/components/BaseCard.vue'
+import { computed, ref } from 'vue'
+import { useRouter } from 'vue-router'
+import RedeemHoloFuelFormStepOne from './RedeemHoloFuelFormStepOne.vue'
+import RedeemHoloFuelFormStepTwo from './RedeemHoloFuelFormStepTwo.vue'
+import { kRoutes } from '@/router'
+
+const router = useRouter()
+
+const props = defineProps<{
+  redeemableHoloFuel: string | number
```

Comment:
Is there a reason we need to ever have these be a number?  We're not doing any calculations ...and this value will also need to able to take up to 18 decimal digit values, so it would be prob be bet practice to keep it as a string to avoid any awkward js rounding / math.

Reply:
Ah, ok, it would be necessary for the value comparison.  

Also... this is a bit of a moot point, but a thought I'd like to confirm nonetheless. -- As far as the currency formatting, I thought as long we set the input type to number and pattern/lang to parse it as a local string we would still get correctly formatted currency strings.  Can you confirm this as true?